src/app/features/shows/my-lists.service.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import camelcaseKeys from 'camelcase-keys';
+import { ShowList } from './models/show-list.model';
+import { MyLists } from './models/my-lists.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MyListsService {
```

Comment:
```suggestion
export class UserListService {
```
đặt tên các thứ tránh để "My" nha e